calico-cloud/threat/web-application-firewall.mdx

```diff
@@ -163,157 +292,28 @@ spec:
   description: 'Test WAF Global Alert'
   severity: 1
   dataSet: waf
-  period: 1m
-  lookback: 1h
+  period: 5m
```

Comment:
This entire section around creating global alerts for WAF can be removed, and it should be removed for 3.18 as well

Reply:
Removed in CE and CC 3.17 and 3.18.

Reply:
Removed